src/Form/Form.js

```diff
@@ -87,11 +87,31 @@ class Form {
 
         this.categorizeInputs()
 
+        this.logFormInfo()
+
         if (shouldAutoprompt) {
             this.promptLoginIfNeeded()
         }
     }
 
+    logFormInfo () {
+        if (!shouldLog()) return
+
+        console.log(`Form type: %c${this.getFormType()} ${this.formAnalyzer.autofillSignal}`, 'font-weight: bold')
+        console.log('Signals: ', this.formAnalyzer.signals)
+        console.log('Wrapping element: ', this.form)
+        console.log('Inputs: ', this.inputs)
+        console.log('Submit Buttons: ', this.submitButtons)
+    }
+
+    getFormType () {
+        if (this.isHybrid) return 'hybrid'
+        if (this.isLogin) return 'login'
+        if (this.isSignup) return 'signup'
+
+        return 'something went wrong'
```

Comment:
NAB: Could we use `unknown`?
```suggestion
        return 'unknown'
```